src/DependencyInjection/CompilerPass/MergeImportedRectorConfigureCallValuesCompilerPass.php

```diff
@@ -35,12 +35,32 @@ public function process(ContainerBuilder $containerBuilder): void
 
     private function completeCollectedArguments(string $serviceClass, Definition $definition): void
     {
-        $configureCallValues = $this->configureCallValuesCollector->getConfigureCallValues($serviceClass);
+        $getConfigureCallValues = $this->configureCallValuesCollector->getConfigureCallValues($serviceClass);
+
+        $configureCallValues = $definition->hasMethodCall(self::CONFIGURE_METHOD_NAME) && $getConfigureCallValues === []
+            ? $this->mergeConfigure($definition->getMethodCalls())
+            : $getConfigureCallValues;
+
         if ($configureCallValues === []) {
             return;
         }
 
         $definition->removeMethodCall(self::CONFIGURE_METHOD_NAME);
         $definition->addMethodCall(self::CONFIGURE_METHOD_NAME, [$configureCallValues]);
     }
+
+    /**
+     * @param array<int, array<int, mixed>> $configuration
+     * @return mixed[]
+     */
+    private function mergeConfigure(array $configuration): array
+    {
+        $mergedConfigure = [];
+
+        foreach ($configuration as $configure) {
+            $mergedConfigure = array_merge_recursive($mergedConfigure, $configure[1][0] ?? []);
```

Comment:
It should be like this:
```suggestion
            if ($methodCall[0] !== 'configure') {
                continue;
            }

            foreach ($methodCall[1] as $configureMethodCall) {
                $mergedConfigure = array_merge_recursive($mergedConfigure, $configureMethodCall);
            }
```

Beause `$configuration` are __all method calls__ we need to filter out anything that is not `configure`.

And then we should loop over `$methodCall[1]` because it can already contain more than 1 call.

Reply:
applied